dev.Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.10-slim
 WORKDIR /app
 
 # Install Poetry
-RUN apt-get update && apt-get install gcc curl -y
+RUN apt-get update && apt-get install gcc g++ curl -y
```

Comment:
g++ required when llama cpp python lib is a dependency

Reply:
LGTM